src/theme/weave.js

```diff
@@ -142,6 +142,16 @@ const weave = {
 
   borderRadius: '4px',
 
+  layer: {
+    front: 1,
+    toolbar: 2,
+    notification: 3,
+    alert: 4,
+    dropdown: 5,
+    tooltip: 6,
+    modal: 7,
+  },
```

Comment:
Nice! Good thought, should modals beat notifications sometimes?  There're prolly not too many cases where this  really bad.

Dropdown over notification?...

Lets give it a go and then tweak it later.

Reply:
lgtm!